test/test_reader.rb

```diff
@@ -0,0 +1,401 @@
+require 'maxmind/db'
+require 'minitest/autorun'
+require 'mmdb_util'
+
+class ReaderTest < Minitest::Test # :nodoc:
```

Comment:
Just wondering what lead you to use `Minitest::Test`. I thought most people used RSpec, but I might be wrong.

Reply:
The Ruby gems guide I read showed it. I didn't look for any alternative. Not really a fan of DSL type testing though. Just another language to learn!